subsys/bluetooth/host/att.c

```diff
@@ -194,6 +202,8 @@ static void bt_att_sent(struct bt_l2cap_chan *ch)
 		chan->sent(chan);
 	}
```

Comment:
Agreed, this looks suspicious, is `chan->sent` being set to NULL elsewhere?

Reply:
That is just there because chan_cb can return NULL, reseting to NULL doesn't make any difference here since chan_send will always attempt to set the callback when called.